examples/raycasting/main.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2016 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"image"
+	"image/color"
+	_ "image/png"
+	"log"
+	"math"
+
+	"github.com/hajimehoshi/ebiten/ebitenutil"
+
+	geo "github.com/paulmach/go.geo"
```

Comment:
Would it be possible to remove this dependency? Or, is this needed for calculation?